handler.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports.get = async (event) => {
+  return {
+    statusCode: 200,
+    body: JSON.stringify(
+      {
+        message: 'Go Serverless v1.0! Your function executed successfully!',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Hindyadav/ked/pullRequest?prid=7538437)
```suggestion
        message: "Go Serverless v1.0! Your function executed successfully!",
```